packages/dapp/src/components/Navbar/SubMenuComponent.tsx

```diff
@@ -20,145 +22,145 @@ import {
-                    style={{ marginLeft: '0px' }}
-                    onClick={() => {
-                      setOpenSubMenu(SubMenuKeys.none);
-                    }}
-                  />
-                  <MenuHeaderLabel>{label}</MenuHeaderLabel>
-                </>
-              </MenuHeaderAppBar>
-            </MenuHeaderAppWrapper>
-            {!!subMenuList.length ? (
-              subMenuList.map((el, index) =>
-                !!el.url ? (
-                  <MenuLinkItem
-                    onClick={() => {
-                      !!el.triggerSubMenu && setOpenSubMenu(el.triggerSubMenu);
-                      !!el.onClick && el.onClick();
+                  {el.prefixIcon}
+                  <Typography
+                    sx={{
+                      maxWidth: 'inherit',
+                      overflow: 'hidden',
+                      textOverflow: 'ellipsis',
                     }}
-                    component="li"
-                    key={`${el.label}-${index}`}
+                    variant={'lifiBodyMedium'}
+                    ml={!!el.prefixIcon ? '12px' : 'inherit'}
+                    mr={!!el.suffixIcon ? '12px' : 'inherit'}
                   >
-                    <MenuItemLabel
-                      variant={
-                        !el.suffixIcon && !el.checkIcon && !el.showMoreIcon
-                          ? 'lg'
-                          : (el.showMoreIcon || el.checkIcon) && el.suffixIcon
-                          ? 'xs'
-                          : 'md'
-                      }
-                    >
-                      <>
-                        {el.prefixIcon}
-                        <Typography
-                          sx={{
-                            maxWidth: 'inherit',
-                            overflow: 'hidden',
-                            textOverflow: 'ellipsis',
-                          }}
-                          variant={'lifiBodyMedium'}
-                          ml={!!el.prefixIcon ? '12px' : 'inherit'}
-                          mr={!!el.suffixIcon ? '12px' : 'inherit'}
-                        >
-                          {el.label}
-                        </Typography>
-                        {el.suffixIcon}
-                      </>
-                    </MenuItemLabel>
-                  </MenuLinkItem>
-                ) : (
-                  <MenuItem
-                    onClick={() => {
-                      !!el.triggerSubMenu && setOpenSubMenu(el.triggerSubMenu);
-                      !!el.onClick && el.onClick();
+                    {el.label}
+                  </Typography>
+                  {el.suffixIcon}
+                </>
+              </MenuItemLabel>
+            </MenuLinkItem>
+          ) : (
+            <MenuItem
+              autoFocus={index > 0 ? true : false}
+              onClick={() => {
+                !!el.triggerSubMenu && onOpenNavbarSubMenu(el.triggerSubMenu);
+                !!el.onClick && el.onClick();
```

Comment:
We can use 

`el.onClick?.();`

Reply:
Never seen this before, but this comes in handy right here. Thanks man!